### DISCUSSION TITLE
Debian 12 packages are missing

### DISCUSSION THREAD

**Discussion by bastelfreak** [puppetlabs/pdk#1329] [Feature Request] [answered]
Debian 12 was released on 10th of June, 2023. Why are there still no pdk packages?

**Comment by david22swan** [comment, marked as answer]
Debian 12 is currently being built and will be brought out as part of the next release.

**Reply by bastelfreak** [reply]
on apt.puppet.com I only found the x86 version for Debian 12. Is it planned to build ARM64 as well (or is that maybe already done and I just didn't find it?)

**Reply by david22swan** [reply]
Looking into the ARM64 for the next release

**Comment by bastelfreak** [comment]
Can you elaborate a bit why it took so long / if this is supposed to take so long for every release?

**Reply by davidsandilands** [reply]
In all honesty there is currently a lack of a clear lifecycle and support statement for the tools like PDK, VSCode and supporting dependencies which can make prioritization and resourcing difficult. We intend to get a full statement so we become more predictable on OS support, major version upgrades and so on,

**Comment by LukasAud** [comment]
Debian 12 packages are currently available for PDK. Closing issue.
